docs/user/entities.rst

```diff
@@ -16,6 +16,26 @@ We often use a pattern like <hostname>(:<port>) to create uniqueness at the host
 
 At the check execution we bind entity properties as default values to the functions executed, e.g. the IP gets used for relative ``http()`` requests.
 
+Format
+------
+
+Generally, ZMON entity is a set of properties that can be represented as a multi-level dictionary. For example:
+
+.. code-block:: json
+
+    {
+        "id":"arbitrary_entity_id",
+        "type":"some_type",
+        "oneMoreProperty":"foo",
+        "nestedProperty": {
+            "subProperty1": "foo",
+            "subProperty2": "bar",            
+        }
+    }
+
+Please note that ``id`` and ``type`` properties are **mandatory**.
```

Comment:
should also add a note that filtering for nested values is currently not supported